api/routes/api.js

```diff
@@ -29,17 +29,11 @@ const init = async () => {
   );
 };
 
-router.get('/', (req, res, next) => res.json({ msg: 'Application Running' }));
+router.get('/', (req, res) => res.json({ msg: 'Application Running' }));
 
 // Register Voter in Blockchain
 // Send OTP to Voter
-router.get('/regVoter/sendOTP', fetchVoter, async (req, res, next) => {
-  if (req.hasRegistered === true) {
```

Comment:
why are we removing this check?

Reply:
If you look carefully then you will find that I have added that check in the middleware itself. I tried to use the same middleware for sending and verifying OTP.